deeplabcut/modelzoo/apis/spatiotemporal_adapt.py

```diff
@@ -0,0 +1,194 @@
+        self.pseudo_threshold = pseudo_threshold
+
+        if modelfolder != "":
+            os.makedirs(modelfolder, exist_ok=True)
+
+            dlc_root_path = os.sep.join(deeplabcut.__file__.split(os.sep)[:-1])
+
+            name_dict = {
+                "supertopview": "supertopview.yaml",
+                "superquadruped": "superquadruped.yaml",
+            }
+
+            self.customized_pose_config = os.path.join(
+                dlc_root_path,
+                "pose_estimation_tensorflow",
+                "superanimal_configs",
+                name_dict[self.supermodel_name],
+            )
+
+        if customized_pose_config != "":
+            # if it's an old modelzoo model, this is also required
+            self.customized_pose_config = customized_pose_config
+
+    def before_adapt_inference(self):
+
+        # save frames have to be on
+        deeplabcut.video_inference_superanimal(
+            [self.video_path],
+            self.supermodel_name,
+            videotype=self.videotype,
+            scale_list=self.scale_list,
+            init_weights=self.init_weights,
+            customized_test_config=self.customized_pose_config,
+        )
+
+        deeplabcut.create_labeled_video(
+            "",
+            [self.video_path],
+            videotype=self.videotype,
+            filtered=False,
+            init_weights=self.init_weights,
+            draw_skeleton=True,
+            superanimal_name=self.supermodel_name,
+            pcutoff=self.pcutoff,
+        )
+
+    def train_without_project(self, pseudo_label_path):
+
+        from deeplabcut.pose_estimation_tensorflow.core.train_multianimal import train
+
+        print("self.customized_pose_config", self.customized_pose_config)
+
+        train(
+            self.customized_pose_config,
+            500,  # displayiters
+            1000,  # saveiters,
+            self.adapt_iterations,  # maxiters
+            modelfolder=self.modelfolder,
+            init_weights=self.init_weights,
+            load_pseudo_label=pseudo_label_path,
+            video_path=self.video_path,
+            pseudo_threshold=0.3,
+        )
+
+    def adaptation_training(self):
+        """
+        There should be two choices, either taking a config, with is then assuming there is a DLC project.
+        Or we make up a fake one, then we use a light way convention to do adaptation
+        """
+
+        # looking for the pseudo label path
+        DLCscorer = "DLC_" + Path(self.init_weights).stem
+        vname = str(Path(self.video_path).stem)
+        video_root = Path(self.video_path).parent
+
+        _, pseudo_label_path, _, _ = deeplabcut.auxiliaryfunctions.load_analyzed_data(
+            video_root, vname, DLCscorer, False, ""
+        )
+
+        self.train_without_project(pseudo_label_path)
+
+    def after_adapt_inference(self):
+
+        pattern = os.path.join(
+            self.modelfolder, f"*/snapshot-{self.adapt_iterations}.index"
```

Comment:
I think this would fail on Windows; we probably need:
```suggestion
            self.modelfolder, "*", f"snapshot-{self.adapt_iterations}.index"
```